ethereum_history_api/oracles/src/script/prepareNoirFixtures.ts

```diff
@@ -43,6 +45,16 @@ for (const chain in FIXTURES) {
           await writeFile(join(modulePath, 'storage_proof.nr'), createStorageProofFixture(stateProof.storageProof));
           fixtureModules.push('storage_proof');
         }
+
+        if (transactionHashes) {
+          const transactionIndexes = transactionHashes.map((hash) => block.transactions.indexOf(hash));
```

Comment:
What happens when tx hash is not to be found within block?

Reply:
I added an assert